ProyectoSemestral3/Assets/Scripts/InventoryScripts/InventoryView.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class InventoryView : MonoBehaviour
+{
+    public Image _itemIcon;
+    public Text _itemName;
```

Comment:
Should be `[SerializeField] private` and have a default value.

Reply:
Done